packages/app-mobile/components/Dropdown.tsx

```diff
@@ -1,31 +1,63 @@
 const React = require('react');
-const { TouchableOpacity, TouchableWithoutFeedback, Dimensions, Text, Modal, View } = require('react-native');
+import {
+	TouchableOpacity, TouchableWithoutFeedback, Dimensions, Text, Modal, View, LayoutRectangle,
+	ViewStyle, TextStyle,
+} from 'react-native';
```

Comment:
Is this done automatically by your editor? I'd rather we keep everything on one line since it takes less vertical space

Reply:
I can change that! (I think I have things configured to auto-wrap at 100 columns).